src/kOS.Safe/Persistence/HarddiskFile.cs

```diff
@@ -1,22 +1,21 @@
-﻿using kOS.Safe.Persistence;
-
-namespace kOS.Safe.Encapsulation
+﻿namespace kOS.Safe.Persistence
 {
     [kOS.Safe.Utilities.KOSNomenclature("VolumeFile", KOSToCSharp = false)]
     public class HarddiskFile : VolumeFile
```

Comment:
I think it was `Harddisk` before and I just kept it that way throughout my changes. So I guess someone decided it was one word.